doc/docs/getting_started.md

```diff
@@ -1,173 +1,559 @@
+
+After [installation and configuration](#installation-and-configuration-1) has been completed, lets start the systemd services.
 
-Once the services are up and running, the logs of the laptop show that the
-Raspberry Pi successfully connects to the laptop:
+Starting the controller and agent on the main node (**laptop**):
 
-```text
-Mar 13 10:20:36 flame.pingoured.fr systemd[1]: Started bluechi-controller.service - BlueChi service orchestrator manager daemon.
-Mar 13 10:20:36 flame.pingoured.fr bluechi-controller[124510]: 10:20:36 INFO     ../src/manager/node.c:602 node_method_register      msg="Registered managed node from fd 8 as 'laptop'"
-Mar 13 10:20:38 flame.pingoured.fr bluechi-controller[124510]: 10:20:38 INFO     ../src/manager/node.c:602 node_method_register      msg="Registered managed node from fd 9 as 'rpi4'"
+```bash
+systemctl start bluechi-controller bluechi-agent
 ```
 
-From there, bluechictl can be used on the laptop to control services running on
-either the laptop itself, or on the Raspberry Pi.
+Switching to the managed node (**raspberry pi**), start the agent:
+
+```bash
+systemctl start bluechi-agent
+```
+
+Once the services are up and running, the journald logs on the **laptop** should show that the both agents have successfully connected to the controller:
+
+```bash
+$ journalctl -u bluechi-controller
+
+Sep 14 14:51:58 mengel systemd[1]: Started BlueChi systemd service controller manager daemon.
+Sep 01 14:51:58 mengel bluechi[3750775]: 2023-09-14 14:51:58,685+0200 INFO        ../src/manager/manager.c:924 manager_start        msg="Starting bluechi 0.5.0"
+Sep 01 14:51:58 mengel bluechi[3750775]: 2023-09-14 14:51:58,928+0200 INFO        ../src/manager/node.c:870 node_method_register    msg="Registered managed node from fd 8 as 'mengel'"
+Sep 01 14:52:02 mengel bluechi[3750775]: 2023-09-14 14:52:02,534+0200 INFO        ../src/manager/node.c:870 node_method_register    msg="Registered managed node from fd 9 as 'pi'"
+...
+```
 
-But before starting services, check which services are running:
+Lets use `bluechictl` on the main node to list all units:
 
 ```bash
-# bluechictl list-units
+# list all units on the laptop
+$ bluechictl list-units mengel
+
 NODE            	|ID                                                     	|   ACTIVE|  	SUB
 ====================================================================================================
-laptop          	|time-sync.target                                       	| inactive| 	dead
-laptop          	|nfs-idmapd.service                                     	| inactive| 	dead
-laptop          	|sys-devices-platform-serial8250-tty-ttyS5.device       	|   active|  plugged
-laptop          	|dev-disk-by\x2did-wwn\x2d0x5001b448b9db9490\x2dpart3.device|   active|  plugged
-laptop          	|podman.socket                                          	|   active|listening
-....
+mengel          	|time-sync.target                                       	| inactive| 	dead
```

Comment:
I'd prefer to use `laptop` or some other common name instead of your specific laptop name

Reply:
Yes, would be better, I agree. Changing it.